### PR TITLE
Text utilities

### PR DESCRIPTION
**Pull request by RaghavendraaReddy**
- please review
- [live CodesandBox](https://t4v61d.csb.app/components/text.html)

### CODE REVIEW

**Comment by SreevidyaNattuva** [comment]
image
Try to use rem instead of px

**Comment by SreevidyaNattuva** [comment]
image
Try to use variables instead of hardcoded values

File: components/avatar/avatar.css [hunk 1]
```diff
@@ -95,6 +95,7 @@ h1{
 }
 .side-bar div{
     margin-top: 10px;
+    margin-left: 25px;
```
Comment (sharmayatendra) [inline components/avatar/avatar.css:98]: use shorthand property

File: components/badge.css [hunk 2]
```diff
@@ -76,12 +76,12 @@
 .status-offline{
     color: red;
     position: absolute;
```
Comment (sharmayatendra) [inline components/badge.css:78]: dont use hard code colors

File: components/badge.css [hunk 3]
```diff
@@ -76,12 +76,12 @@
...
-    left: 755px;
-    top:450px;
+    left: 850px;
+    top:430px;
     }
 .status-available{
     color: green;
     position: absolute;
-    left: 755px;
```
Comment (sharmayatendra) [inline components/badge.css:85]: same as above

File: components/text.css [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+ }
+ .container{
+     display: flex;
+     background-color: lightblue;
+     padding: 1rem;
+     box-shadow: 0px 3px whitesmoke;
+ }
+ #logo{
+     font-size: 55px;
+     margin-left: 45px;
```
Comment (sharmayatendra) [inline components/text.css:12]: use rem unit

File: components/text.css [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+     color: black;
+     text-decoration: none;
+ }
+ .footer{
+     display: flex;
+     gap:1rem;
+     justify-content: center;
+     align-items: center;
+     background-color: black;
+     padding: 70px 40px;
```
Comment (sharmayatendra) [inline components/text.css:32]: use rems & color variables

File: components/text.css [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+ .side-bar a{
+     text-decoration: none;
+     color: black;
+     font-size: 22px;
+     
+ }
+ .side-bar div{
+     margin-top: 10px;
+     margin-left: 25px;
+ }
```
Comment (sharmayatendra) [inline components/text.css:55]: same as above

**Review by sharmayatendra — APPROVED** [review]
look upon the suggested changes everywhere

**Review by Rajitha04 — COMMENTED** [review]
Try to use variable colors, rest is fine

**Pull request merged**